src/components/WebexMemberRoster/WebexMemberRoster.jsx

```diff
@@ -23,12 +23,32 @@ import useMemberships from '../hooks/useMemberships';
  * @returns {object} JSX of the component
  */
 export default function WebexMemberRoster({destinationID, destinationType}) {
-  const memberships = useMemberships(destinationID, destinationType);
-  const members = memberships.map(
+  const members = useMemberships(destinationID, destinationType);
+
+  const renderMembers = (members) => (members || []).map(
     (member) => <WebexMember key={member.personID} personID={member.personID} />,
   );
 
-  return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{members}</div>;
+  if (destinationType !== 'meeting') {
+    return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{renderMembers(members)}</div>;
+  }
+
+  const isInMeetingMembers = members.filter(membership => membership.isInMeeting);
```

Comment:
let's use `inMeetingMembers` and `notInMeetingMembers` (without the `is` prefix) so that they don't look like booleans. And the current item in filter should be called `member`.